test/trip-test.js

```diff
@@ -32,6 +34,10 @@ describe('Trip', function() {
       expect(pastTrip.status).to.equal("approved")
       expect(pastTrip.suggestedActivities).to.deep.equal([])
     });
+    it('should default to placeholder data if data is missing or not provided', () => {
```

Comment:
This is a good test. Simple and easy to understand. Good to cover these sad paths